### ISSUE TITLE
I hear echo & background noise on the DEMO wavs

### ISSUE THREAD

**Issue by creotiv** [keonlee9420/PortaSpeech#15]
This can happen because of blurring of mel-spectrogram(in most cases start of the sound should have sharp edge), which can happen because bad attention. I could recommend to try using Diagonal guided attention (DGA) during the training(https://arxiv.org/abs/1710.08969)

**Comment by keonlee9420** [comment]
Hi @creotiv , thanks for your suggestion. Unfortunately, I'd already tried several methods including DGA to improve attention as described in [here](https://github.com/keonlee9420/PortaSpeech/issues/3#issuecomment-950599642), but the output quality is degraded even with the improved (strong) attention. This might be due to the model architecture or the loss design so I'm trying to change them now.

**Comment by michaellin99999** [comment]
> Hi @creotiv , thanks for your suggestion. Unfortunately, I'd already tried several methods including DGA to improve attention as described in [here](https://github.com/keonlee9420/PortaSpeech/issues/3#issuecomment-950599642), but the output quality is degraded even with the improved (strong) attention. This might be due to the model architecture or the loss design so I'm trying to change them now.

Hey keonlee we would like to help collaborate to find a solution. What other methods have you tried? We tried using clustering and embedding of training data and output and find we aren’t able to clearly identify the problem. the words that echo and regular output words have similar results

**Comment by keonlee9420** [comment]
> > Hi @creotiv , thanks for your suggestion. Unfortunately, I'd already tried several methods including DGA to improve attention as described in [here](https://github.com/keonlee9420/PortaSpeech/issues/3#issuecomment-950599642), but the output quality is degraded even with the improved (strong) attention. This might be due to the model architecture or the loss design so I'm trying to change them now.
> 
> Hey keonlee we would like to help collaborate to find a solution. What other methods have you tried? We tried using clustering and embedding of training data and output and find we aren’t able to clearly identify the problem. the words that echo and regular output words have similar results

Hi @michaellin99999 , thanks for your attention! I just updated the repo with the fixation (there was a bug in code, see this [commit](https://github.com/keonlee9420/PortaSpeech/commit/0dd91701685e874398cbdf3e7f3fd6ce0dbee661)). I've also applied several techniques to improve the alignment including DGA. So please check out the note section in README for the details. Also, try the new pre-trained model and demo :) Any suggestions are always welcome!

**Comment by keonlee9420** [comment]
Close due to inactivity
